apps/ledger-live-mobile/src/actions/general.ts

```diff
@@ -90,12 +91,8 @@ export function useCleanCache() {
   const dispatch = useDispatch();
   const { wipe } = useCountervaluesPolling();
   return useCallback(async () => {
-    dispatch({
-      type: "CLEAN_CACHE",
-    });
-    dispatch({
-      type: "LEDGER_CV:WIPE",
```

Comment:
Do not we need LEDGER_CV:WIPE anymore?

Reply:
it was no longer used. it's a relicat from the past.